src/martialarts.cpp

```diff
@@ -65,19 +65,13 @@ void ma_requirements::load( JsonObject &jo, const std::string & )
 
 void ma_technique::load( JsonObject &jo, const std::string &src )
 {
-    optional( jo, was_loaded, "name", name, translated_string_reader );
-    optional( jo, was_loaded, "description", description, translated_string_reader );
+    optional( jo, was_loaded, "name", name, "" );
+    optional( jo, was_loaded, "description", description, "" );
```

Comment:
Are you sure it makes sense to make description and name default to an empty string? Why would somebody want empty strings there? If the entry does not exist in JSON, it's almost always an oversight / misspelling and the game should complain about that.

Note that removing the empty default string will still allow JSON entries that have an empty string as that entry, it just needs to be written explicitly.

Reply:
So shall I give a `debugmsg` if `name` field doesn't exist?

Reply:
You better call `mandatory` instead of `optional` like in other places when `name` and `description` are read, i.e.:

https://github.com/CleverRaven/Cataclysm-DDA/blob/98926b679a8c1122d38489afb315a47ff693ea9e/src/martialarts.cpp#L125-L126